apps/voting/test/voting.js

```diff
@@ -174,189 +139,234 @@ contract('Voting App', accounts => {
             assert.equal(await voting.minAcceptQuorumPct(), 1, 'should have changed acceptance quorum')
         })
 
-        it('fails changing minimum acceptance quorum to zero', async () => {
-            return assertRevert(async () => {
-                await voting.changeMinAcceptQuorumPct(0)
-            })
-        })
-
         it('fails changing minimum acceptance quorum to greater than min support', async () => {
             return assertRevert(async () => {
                 await voting.changeMinAcceptQuorumPct(neededSupport.plus(1))
             })
         })
 
-        context('creating vote', () => {
-            let script, voteId
+    })
+
+    for (const decimals of [0, 2, 10, 18, 36]) {
+        context('normal token supply, ' + decimals + ' decimals', () => {
+            const holder20 = accounts[0]
+            const holder29 = accounts[1]
+            const holder51 = accounts[2]
+            const nonHolder = accounts[4]
+
+            const neededSupport = pct16(50)
+            const minimumAcceptanceQuorum = pct16(20)
 
             beforeEach(async () => {
+                token = await MiniMeToken.new(NULL_ADDRESS, NULL_ADDRESS, 0, 'n', decimals, 'n', true) // empty parameters minime
+
+                await token.generateTokens(holder20, 20)
```

Comment:
Assigned balances should be multiplied by `10 ^ decimals`, otherwise we are just testing the same every time. You can use logic similar to the one in the `pct16` helper